### PR TITLE
Open Source Rubocop-Airbnb gem into the ruby repository

### LINKED ISSUE #94: Rubocop ruleset?
Is there a rubocop ruleset for this coding style similar to your eslint for js?

### LINKED ISSUE #112: License?
Hello, 

I'd like to use this style guide as a basis to start our own for my team, but there's no license information and I don't want to step on toes. Can someone please specify how this is made available? 

Thanks!

### PR DESCRIPTION
**Pull request by robotpistol**
This PR adds the rubocop-airbnb gem implementation to the ruby style guide. The hope is that over time the 2 will become increasingly in sync as we continue to enable rules.  Thank you all for your patience in me getting this released publicly.  I look forward to seeing the whole thing evolve.

 @airbnb/ruby-styleguide-maintainers 
 @airbnb/ruby-styleguide-owners  

Closes #94. Closes #112.

### CODE REVIEW

File: rubocop-airbnb/.gitignore [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.*.swp
+.\#*
+\#*#
+node_modules
+npm-debug.log
+
+rubocop-airbnb*.gem
+
+# IDEs
+.idea
```
Comment (BMorearty) [inline rubocop-airbnb/.gitignore]: It's not the end of the world if they are in here I recommend people put environment-specific ignore files such as editor files in their global .gitignore, not in each repo. The same advice applies to some of the other lines in this file, such as `#*#` and `.\#*` and `.*.swp`.

File: rubocop-airbnb/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+<!--- required --->
+
+#### Reviewers
+
+<!--- required --->
+
+#### JIRA Task
+
+<!--- optional --->
```
Comment (BMorearty) [inline rubocop-airbnb/PULL_REQUEST_TEMPLATE.md]: JIRA Task doesn't make sense in open source

File: rubocop-airbnb/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## Screenshots/GIF (if visual change)
+
+<!--- optional --->
```
Comment (BMorearty) [inline rubocop-airbnb/PULL_REQUEST_TEMPLATE.md]: There probably aren't any screenshots?

File: rubocop-airbnb/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# RuboCop Airbnb
+
+Airbnb specific analysis for [RuboCop](https://github.com/bbatsov/rubocop).
+
+It contains Airbnb's internally used configuration for
+[RuboCop](https://github.com/bbatsov/rubocop) and
+[RuboCop RSpec](https://github.com/backus/rubocop-rspec). It also includes custom
```
Comment (BMorearty) [inline rubocop-airbnb/README.md]: sentence wasn't finished?

File: rubocop-airbnb/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (robotpistol) [reply rubocop-airbnb/PULL_REQUEST_TEMPLATE.md]: ah thought i'd removed this.

File: rubocop-airbnb/.gitignore (hunk 1 above)
Comment (ljharb) [reply rubocop-airbnb/.gitignore]: Oh also this file should be moved up to the root

Comment (robotpistol) [reply rubocop-airbnb/.gitignore]: Cleaning up the .gitignore before merging.

**Pull request merged**